lxl-web/src/lib/components/Search.svelte

```diff
@@ -34,11 +35,13 @@
 	}
 </script>
 
-<form class="w-full" action="find" on:submit={handleSubmit}>
+<form class="relative w-full" action="find" on:submit={handleSubmit}>
 	<!-- svelte-ignore a11y-autofocus -->
 	<input
 		id="main-search"
-		class="h-12 w-full rounded-full border-t-accent-dark/32 text-secondary sm:h-14"
+		class="h-12 w-full rounded-full pr-12 text-secondary shadow-accent-dark/32 focus:shadow-search-focus focus:outline
+			focus:outline-8 focus:outline-accent-dark/8 sm:h-14
+			sm:pr-28"
```

Comment:
I would prefer the default browser outline styling as the outlines from the Figma sketches are extremely subtle. I feel that we loose more (accessiblity) than we gain from the custom styling.

Current custom outline (which I think is too subtle):
![Skärmavbild 2024-05-13 kl  13 41 18](https://github.com/libris/lxlviewer/assets/10220360/8c6a9260-a726-4044-b9d4-15768276c45c)

Default browser outline in Firefox:
![Skärmavbild 2024-05-13 kl  13 41 01](https://github.com/libris/lxlviewer/assets/10220360/7ead047f-ef07-4fb7-8f3e-f41a90ca0072)

Reply:
I'm also ok with having a shadow outline if has more contrast / is more visible.

Reply:
<img width="444" alt="Skärmavbild 2024-05-13 kl  14 00 36" src="https://github.com/libris/lxlviewer/assets/22232928/55a7f90c-7d6e-424a-a34b-97ff8d19fe9a">

Revisiting the Figma design now I realize the outline is probably meant as hover state (similarly, the buttons also have hover outline). Maybe a bit unconventional for an input field. I'm ok with skipping it now and going for a general, fully visible/accessible focus ring later.

Reply:
Turned the contrast up a notch. Visible enough?
<img width="1496" alt="Skärmavbild 2024-05-14 kl  10 51 11" src="https://github.com/libris/lxlviewer/assets/22232928/f972d061-6723-4c4e-83f6-ac1284d7ee66">

Reply:
Much better, I'm fine with it :)

The only problem with adding a custom outline in one place is that missing customized outlines on other elements gets more glaring/noticeable. 😀 But that should not hinder us!

Reply:
Yep, but that's the beta feel :)